hilti/runtime/CMakeLists.txt

```diff
@@ -161,8 +161,3 @@ target_compile_options(hilti-rt-configuration-tests PRIVATE $<$<CONFIG:Debug>:-O
 target_link_options(hilti-rt-configuration-tests PRIVATE $<$<CONFIG:Debug>:-O0>)
 target_link_libraries(hilti-rt-configuration-tests PRIVATE $<IF:$<CONFIG:Debug>,hilti-rt-debug,hilti-rt> doctest)
 add_test(NAME hilti-rt-configuration-tests COMMAND ${CMAKE_BINARY_DIR}/bin/hilti-rt-configuration-tests)
-
```

Comment:
Why's this going out?

Reply:
This shouldn't have ended up in this PR, I'll fix it.